common/helpers/front_end.py

```diff
@@ -20,6 +20,8 @@ def arg_string(idx, key, value):
 
 
 def section_url(section, args_dict=None):
+    if section == 'Blog':
+        return settings.BLOG_URL
```

Comment:
We don't want to put in special cases like this, otherwise if we need to do more external redirects in the future things will become quite messy.